### DISCUSSION TITLE
关于alist Windows端使用vbs实现自启动错误的疑问

### DISCUSSION THREAD

**Discussion by dreamrfy** [AlistGo/alist#3451] [Q&A] [answered]
我根据alis文档指引写了启动的vbs，当我在alist.exe文件夹里面双击是可以正常的运行该vbs，但是我把该vbs创建一个快捷方式，并把该快捷方式剪切到Windows开始文件夹下，再次双击就会报错。
提示错误为：

脚本:	D:\快速启动\alist\启动.vbs
行:	3
字符:	1
错误:	系统找不到指定的文件。 
代码:	80070002
源: 	(null)

该vbs代码为
Dim ws
Set ws = Wscript.CreateObject("Wscript.Shell")
ws.run "alist.exe server",vbhide
Wscript.quit

请问各位大佬这个应该怎么处理啊？

**Comment by anwen-anyi** [comment, marked as answer]
很明显你实际的vbs脚本没有和Alist执行文件在一个文件夹，
脚本找不到执行文件可不得报错嘛，
或者你的Alist执行程序文件叫其他名字和脚本里面的不一样

>│  alist.exe
>│ 启动.vbs
>│ 停止.vbs
>├─data
>│  │  config.json
>│  │  data.db
>│  │  data.db-shm
>│  │  data.db-wal

**Comment by PennHui666** [comment]
Dim ws
Set ws = CreateObject("Wscript.Shell") 
ws.run "cmd /c D:\alist-windows-amd64\alist.exe server",vbhide
Wscript.quit

将vbs中的代码改成这段，其中的alist.exe路径改成自己安装文件夹的路径即可正常使用。不会出现密码错误和不启动的情况。
